pipeline: outputs: azure_blob: add log_key to outputs (#1540)
fluent/fluent-bit-docs: pipeline/outputs/azure_blob.md

| sas\_token | Specify the Azure Storage shared access signatures to authenticate against the service. This configuration property is mandatory when `auth_type` is `sas`. |  |
| container\_name | Name of the container that will contain the blobs. This configuration property is mandatory |  |
| blob\_type | Specify the desired blob type. Fluent Bit supports `appendblob` and `blockblob`. | appendblob |

> | log\_key | By default, the whole log record will be sent to blob storage. If you specify a key name with this option, then only the value of that key will be sent. |  |

Reviewer: ```suggestion
| log\_key | By default, the entire log record is sent to blob storage. If you specify a key name with this option, only the value of that key is sent. |  |
```
